srtcp.go

```diff
@@ -20,6 +22,16 @@ func (c *Context) decryptRTCP(dst, encrypted []byte) ([]byte, error) {
 		return out, nil
 	}
 
+	srtcpIndexBuffer := encrypted[tailOffset : tailOffset+srtcpIndexSize]
+
+	index := binary.BigEndian.Uint32(srtcpIndexBuffer) &^ (1 << 31)
+	ssrc := binary.BigEndian.Uint32(encrypted[4:])
+
+	markAsValid, ok := c.srtcpReplayDetector.Check(uint64(index))
+	if !ok {
+		return nil, errDuplicated
```

Comment:
Do you know what the UX for `pion/webrtc` will be like right now? Does `mux` just print this error, or does it `Close`/tear down everything?

Reply:
This error is returned to the user code only if `srtp.Context` is directly used. `pion/webrtc` uses srtp via `srtp.SessionSRT(C)P`. `errDuplicated` is consumed at https://github.com/pion/srtp/blob/a61c2acb40ffcd747f4e72c06dce8fa658082d9b/session.go#L132-L134